dlk/python/dlk/templates/Makefile.tpl

```diff
@@ -153,7 +153,7 @@ lib_x86:           CXX = g++
 lib_x86:           FLAGS += $(INCLUDES) -O3 -std=c++0x -fPIC -fvisibility=hidden -pthread -g
 
 lib_aarch64:       CXX = aarch64-linux-gnu-g++
-lib_aarch64:       FLAGS += $(INCLUDES) -O3 -std=c++0x -fPIC -DUSE_NEON -fvisibility=hidden -pthread -g
+lib_aarch64:       FLAGS += $(INCLUDES) -O3 -std=c++0x -fPIC -fvisibility=hidden -pthread -g
```

Comment:
The indent looks messy for me 😭😭😭 
<img width="427" alt="screen shot 2019-02-20 at 15 06 03" src="https://user-images.githubusercontent.com/138891/53074392-fa42e800-352d-11e9-98e5-61ffe04dce9e.png">

Reply:
OK. I will fix it.

Reply:
😄